### PR TITLE
Clarify support and option how to support Tycho

### PR DESCRIPTION
**Pull request by laeubi**
@akurtakov @mickaelistria WDYT?

Please 

- Approve PR if you like it
- Request changes if you think something should be changed
- Write a comment if you think it is not suitable at all

Preview page: https://github.com/eclipse-tycho/tycho/blob/efea58a145b5c7b859ececeefd164f344a01ffb0/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,42 @@ with Maven.
...
-Sponsoring individual committer if you want to help with development of Tycho itself or the Eclipse Foundation in general is welcome. You can do that at:
+# Getting support
+
+Just get in contact with us through the [discussions](https://github.com/eclipse-tycho/tycho/discussions) and share your [ideas](https://github.com/eclipse-tycho/tycho/discussions/new) or report [issues](https://github.com/eclipse-tycho/tycho/issues).
+
+Please bear in mind that this project is almost entirely developed by volunteers and is not a product you contracted for. As a result, the contributors may not be able to look into some support requests. As per Eclipse Development Process, the committers are committed to review incoming code contributions though. If you do not provide the fix/implementation yourself (or pay someone to do it for you), the bug might never get fixed. If it is a serious bug, other people than you might care enough to provide a fix. As a consequence of all that, the only sure way to ensure some issue gets fixed or some feature gets implemented is that you contribute it yourself and convince some contributor that the change you submit is in the best interested of the project.
```
Comment (mickaelistria) [inline README.md:33]: I would skip the term "volunteer", and just write that  "Please bear in mind that this is a community project, and not a product you contracted support for." Volunteer usually is a synonymous for "doing it for free" in French at least; and to be clear, I'm personally not working on Tycho for free nor will I ever do I believe; my employer does pay me for that as it fits in some higher level goals.
I would also replace "the contributors may not be able to look into support requests" as it's not a matter of ability only, but also and mainly of will. I would say "Some contributors may or may not look at your support requests, at their own convenience".

Comment (laeubi) [reply README.md]: I rephrased and shorten this section now!

**Comment by mickaelistria** [comment]
Ok with me

**Review by akurtakov — APPROVED** [review]

**Comment by akurtakov** [comment]
I'm merging it right now.

**Pull request merged**